src/routes/index.svelte

```diff
@@ -22,16 +22,16 @@
         },
     ]
 
-    let history = []
     let dates = []
     let valueSet = []
-
+    let history = new Map()
     let currentDay = 0
     let currentStock = ''
 
-    function getChartSeries(history) {
-        for (let i = 0; i < history.length; i++) {
-            valueSet[i] = history[i].close.slice(0, currentDay + 1)
+    function getChartSeries(stockData) {
+        valueSet = []
+        for (let value of stockData.values()) {
+            valueSet = [...valueSet, value.close.slice(0, currentDay + 1)]
```

Comment:
```suggestion
            valueSet.push(value.close.slice(0, currentDay + 1)
```
Since valueSet = [] will trigger the reactivity